### PR TITLE
Cluster changes

### PR DESCRIPTION
**Pull request by jastBytes**
Make it possible to update cluster name. Drop display name for clusters.

### CODE REVIEW

File: cmd/monoctl/update/cluster.go [hunk 1]
```diff
@@ -65,16 +67,28 @@ func NewUpdateClusterCmd() *cobra.Command {
 				}
 			}
 
+			if newName != "" && newName != name {
+				prompt := promptui.Prompt{
```
Comment (HaniAlshikh) [inline cmd/monoctl/update/cluster.go:71]: can you please wrap it like [for example](https://github.com/finleap-connect/monoctl/blob/ff804fdf96fafd715830c6408a5ca7f48c899e72/internal/prompt/prompt.go#L15) or was it intentional to not do so?

File: go.mod [hunk 2]
```diff
@@ -4,22 +4,22 @@ go 1.19
 
 require (
 	github.com/briandowns/spinner v1.19.0
-	github.com/finleap-connect/monoskope v0.4.15
+	github.com/finleap-connect/monoskope v0.5.2-0.20221027115350-9fd1a29a5af2
```
Comment (HaniAlshikh) [inline go.mod:7]: if you are planning to release we should update this.

**Review by HaniAlshikh — CHANGES_REQUESTED** [review]

File: cmd/monoctl/create/cluster.go [hunk 3]
```diff
@@ -77,15 +69,17 @@ func NewCreateClusterCmd() *cobra.Command {
...
 			})
 		},
 	}
 
 	flags := cmd.Flags()
 
-	flags.StringVarP(&name, "name", "n", "", "Name of the cluster")
-	flags.StringVarP(&displayName, "display-name", "d", "", "Display name of the cluster")
+	flags.StringVarP(&caCertBundleFile, "ca-filepath", "c", "", "Path to the file containing the CA certificate bundle of the cluster in PEM format.")
+	flags.StringVarP(&caCertBundleFile, "api-server-address", "a", "", "Address of the KubeAPIServer of the cluster.")
```
[the remark this answers is not in the document]
Comment (jastBytes) [reply cmd/monoctl/create/cluster.go]: Solved

File: go.mod (hunk 2 above)
Comment (jastBytes) [reply go.mod:7]: agreed

File: cmd/monoctl/update/cluster.go (hunk 1 above)
Comment (jastBytes) [reply cmd/monoctl/update/cluster.go]: updated

**Review by HaniAlshikh — APPROVED** [review]

**Review by HaniAlshikh — APPROVED** [review]

**Review by HaniAlshikh — APPROVED** [review]

**Pull request merged**